### PR TITLE
Log vmem requests

### PR DESCRIPTION
**Pull request by itkovian**

### CODE REVIEW

File: bin/myshowq.py [hunk 1]
```diff
@@ -87,7 +82,7 @@ def read_cache(owner, showvo, running, idle, blocked, path):
     if not showvo:
         for user in res.keys():
             if not user == owner:
-                #del res[user]
+                # del res[user]
```
Comment (stdweird) [inline bin/myshowq.py]: remove if unused

File: bin/submitfilter.py [hunk 2]
```diff
@@ -71,21 +71,30 @@ def make_new_header(sf):
...
+            make("-m", "n"),
         ])
 
-    #    vmem: add default when not specified
-    if not 'vmem' in state['l']:
-        (ppp, vpp) = get_cluster_mpp(state['_cluster'])
-        vmem = vpp * ppn
+    current_user = pwd.getpwuid(os.getuid()).pw_name
+
+    # pvmem: add default when not specified
```
Comment (stdweird) [inline bin/submitfilter.py]: not pvmem

**Comment by itkovian** [comment]
Jenkins please test this

**Comment by stdweird** [comment]
test this please

**Comment by stdweird** [comment]
test this please

**Comment by stdweird** [comment]
@itkovian jenkins was not properly configured

File: setup.py [hunk 3]
```diff
@@ -36,7 +36,7 @@
         'vsc-ldap >= 1.3.4',
         'vsc-ldap-extension >= 1.10',
         'vsc-utils >= 1.4.6',
-        'pbs_python >= 4.6', # don't use installs from pypi
+#        'pbs_python >= 4.6', # don't use installs from pypi
```
Comment (stdweird) [inline setup.py]: remove this again

**Comment by stdweird** [comment]
Fixed in #73

**Pull request closed without merging**